plugins/scaffolder-node/src/tasks/types.ts

```diff
@@ -119,6 +124,8 @@ export interface TaskContext {
   emitLog(message: string, logMetadata?: JsonObject): Promise<void>;
 
   getWorkspaceName(): Promise<string>;
+
+  getInitiatorCredentials(): Promise<BackstageCredentials>;
```

Comment:
@benjdlambert is this an output type? The purpose of the test helper was to start reducing the impact of adding fields here in test code. Did we settle on whether we can break this in this way as part of the retry work?

Reply:
It's the type that defines the `ctx` input to your `handler`:

https://github.com/backstage/backstage/blob/19d3cb9d1283d3e806ae3d281004374baa140788/plugins/scaffolder-backend-module-github/src/actions/githubActionsDispatch.ts#L86

The breakages are only in tests when people are mocking them out, so feeling that it's fine to break this type and we push people into using the helper instead which abstracts the production of the context handler instead.

Reply:
I might tidy up `createMockActionContext`, just noticed now that the types can be simplified.